examples/webgl2_multiple_rendertargets.html

```diff
@@ -246,6 +245,13 @@
 
 				renderTarget.samples = parameters.samples;
 
+				// safari prior 15.5 needs the renderer to be cleared()
+				// if ( parameters.samples > 0 ) {
+
+					// renderer.clear();
+
+				// }
+
```

Comment:
This also affected scenes that used `antialias` on the renderer, with `gl.clear()` being a workaround that accepted any combination of bits -- even simply zero.

Since being patched, I think we can direct users to keep their browsers up-to-date. iOS is a bit too unstable for comfort.